src/guillotine/fetchContent.ts

```diff
@@ -33,6 +30,7 @@ import {combineMultipleQueries} from './combineMultipleQueries';
 import {fetchContentData} from './fetchContentData';
 import {applyProcessors} from './applyProcessors';
 import {createMetaData} from './createMetaData';
+import {draftMode, headers} from 'next/headers';
```

Comment:
Why did you move the import from next to the bottom? I typically try to order imports by distance and alphabetically:

type npmmodules
type /absolute
type ../upstream
type ./downstream
type ./same folder

npmmodules
/absolute
../upstream
./downstream
./same folder

Reply:
It's not me, it's IntelliJ 👉

Reply:
Then I think your IntelliJ makes a lot of unnecessary changes.

Reply:
Ok, I turned off import sorting in IntelliJ, but this file doesn't look particularly sorted to me anyway ;)

Reply:
But in general, I don't know what logic one might use for sorting, so I think it's better to have some automatic sorting convention